invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/detail.html

```diff
@@ -214,12 +214,7 @@ <h2>
 
 {% macro list_vocabulary_values(field, values, field_cfg) %}
   {% if values.title_l10n is defined %}
-    {% set search_url = field | custom_fields_search(values.title_l10n, field_cfg) %}
-    {% if search_url %}
-      <a href="{{ search_url }}">{{ values.title_l10n | safe }}</a>
-    {% else %}
-      {{ values.title_l10n }}
-    {% endif %}
+    {{ values.title_l10n }}
```

Comment:
Why do we remove the search link from all vocabularies?

Reply:
It is only for the custom_fields, this macro is being called by `show_section_custom_fields` only, maybe better to rename this macro to `list_custom_fields_vocabulary_values`?
The link redirects to a query string search which doesn't work in the records search, [see example](https://repository.cern/search?q=custom_fields.cern%5C%3Aprogrammes%3ACERN%20Doctoral%20Student%20Program&l=list&p=1&s=10&sort=bestmatch)